legaltext/actions.py

```diff
@@ -0,0 +1,57 @@
+import io
+import zipfile
+
+from django.http import HttpResponse
+from django.utils import timezone
+from django.utils.timezone import localtime
+from django.utils.translation import ugettext_lazy as _
+
+
+def export_legal_text_version_action(description=None):
```

Comment:
I would prefer a class with classmethod as "action" handler. nested functions are not what I would use here.